### DISCUSSION TITLE
当数据中存在多个具有相同行列且值相同时，s2会忽略其余相同值的行数据，导致数值列合计信息不对

### DISCUSSION THREAD

**Discussion by acduan** [antvis/S2#2820] [Q&A] [answered]
有相同行列且值相同时，数值合计纸和笔应该分别为2，合计数应该为4，但结果显示的为2
![image](https://github.com/user-attachments/assets/21e86e1a-5128-4c13-916c-136967e7b790)
![image](https://github.com/user-attachments/assets/937beca6-68d4-4d97-b99f-277483dc97b9)

`<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8" />
	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
	<title>浏览器引入</title>
</head>

<body>
	<div id="container" />
</body>
<script src="https://unpkg.com/@antv/s2@2.0.0-next.25/dist/index.min.js"></script>
<script>
	var datas = [
		{
			province: "浙江",
			city: "杭州",
			type: "笔",
			price: "1"
		},
		{
			province: "浙江",
			city: "杭州",
			type: "笔",
			price: "1"
		},
		{
			province: "浙江",
			city: "舟山",
			type: "纸张",
			price: "1"
		},
		{
			province: "浙江",
			city: "舟山",
			type: "纸张",
			price: "1"
		},
	]

	const s2DataConfig = {
		fields: {
			rows: ["province", "city"],
			columns: ["type"],
			values: ["price"]
		},
		data: datas
	};

	const s2Options = {
		width: 800,
		height: 480,
		totals: {
			row: {
				showGrandTotals: true,
				showSubTotals: { always: false },
				reverseGrandTotalsLayout: true,
				reverseSubTotalsLayout: true,
				subTotalsDimensions: ["province"],
				calcGrandTotals: {
					aggregation: 'MIN',
				},
				calcSubTotals: {
					aggregation: 'SUM',
				},
				subTotalsLabel: '小计'
			},
			col: {
				showGrandTotals: true,
				showSubTotals: { always: false },
				reverseGrandTotalsLayout: true,
				reverseSubTotalsLayout: true,
				subTotalsDimensions: ["province"],
				calcGrandTotals: {
					aggregation: 'SUM',
				},
				calcSubTotals: {
					aggregation: 'SUM',
				},
			},
		},
	};

	const container = document.getElementById("container");
	const s2 = new S2.PivotSheet(container, s2DataConfig, s2Options);
	s2.render();
</script>

</html>`

**Comment by lijinke666** [comment, marked as answer]
可参考: https://github.com/antvis/S2/issues/2767#issuecomment-2164209195
